iocore/net/QUICNet.cc

```diff
@@ -77,6 +77,7 @@ QUICPollCont::_process_packet(QUICPollEvent *e, NetHandler *nh)
   }
 
   // Note: We should free QUICPollEvent here since vc could be freed from other thread.
+  p->free();
```

Comment:
I don't think we can free the `p` here because QUICNetVC doesn't immediately process packets in `handle_receiv ed_packet()`. It should be freed on QUICNetVC side.

Reply:
Actually the Quiche version may process packets immediately, but it'd break the original version.

Reply:
`quic_conn_recv()` get called right before this, does that process the packet?

`QUICPollCont::_process_packet() > QUICNetVConnection::handle_received_packet() > quiche_conn_recv()`

Reply:
I found #ifdef. This code is just for quiche and no problem.